docs/updates/release-notes.md

```diff
@@ -12,6 +12,36 @@ import CloudBadge from '@site/src/components/CloudBadge';
 
 # Latest Release Notes
 
+## 4.0.0 - 2023-12-01 - Connections Page, Disconnect Profile and Change Provider
```

Comment:
@CanKarabulut We will go on with version `3.10.0`. Currently there is no reason for us to bump major version. :wink:

Reply:
fixed.